turbo/rpchelper/filters.go

```diff
@@ -67,22 +70,26 @@ func New(ctx context.Context, ethBackend ApiBackend, txPool txpool.TxpoolClient,
 		pendingHeadsStores: concurrent.NewSyncMap[HeadsSubID, []*types.Header](),
 		pendingTxsStores:   concurrent.NewSyncMap[PendingTxsSubID, [][]types.Transaction](),
 		logger:             logger,
+		config:             config,
 	}
 
 	go func() {
 		if ethBackend == nil {
 			return
 		}
+		activeSubscriptionsLogsClientGauge.With(prometheus.Labels{clientLabelName: "ethBackend_Events"}).Inc()
 		for {
 			select {
 			case <-ctx.Done():
+				activeSubscriptionsLogsClientGauge.With(prometheus.Labels{clientLabelName: "ethBackend_Events"}).Dec()
 				return
 			default:
 			}
 
 			if err := ethBackend.Subscribe(ctx, ff.OnNewEvent); err != nil {
 				select {
 				case <-ctx.Done():
+					activeSubscriptionsLogsClientGauge.With(prometheus.Labels{clientLabelName: "ethBackend_Events"}).Dec()
```

Comment:
in `kv_interface.go` we have pre-allocated metrics with labels:
```
DbCommitSync   = metrics.GetOrCreateSummary(`db_commit_seconds{phase="sync"}`)
```
maybe it's better to pre-alloc them instead of call `.With` at runtime?